external-resolver/src/main/resources/application.conf

```diff
@@ -1,10 +1,6 @@
 akka {
   loggers = ["akka.event.slf4j.Slf4jLogger"]
   loglevel = "INFO"
-  http.server.request-timeout = 2 minutes
-  http.server.request-timeout = ${?AKKA_HTTP_REQUEST_TIMEOUT}
-  http.server.idle-timeout = 2 minutes
-  http.server.idle-timeout = ${?AKKA_HTTP_IDLE_TIMEOUT}
```

Comment:
True, but until we actually have a need to increase it, it seems like it would only hurt performance on misbehaving connections. Seems better to just readd it if we need it?